lib/miq_automation_engine/service_models/miq_ae_service_container_deployment.rb

```diff
@@ -0,0 +1,255 @@
+require 'net/scp'
+
+module MiqAeMethodService
+  class MiqAeServiceContainerDeployment < MiqAeServiceModelBase
+    expose :container_deployment_nodes, :association => true
+    expose :deployed_ems, :association => true
+    expose :deployed_on_ems, :association => true
+    expose :automation_task, :association => true
+    expose :roles_addresses
+    expose :container_nodes_by_role
+    expose :ssh_auth
```

Comment:
Remove the `expose` line.

Instead create a ssh_auth private method that returns the active record object.

```ruby
def ssh_auth
  object_send(:ssh_auth)
end
private :ssh_auth
```